### ISSUE TITLE
Provide more feedback when logging a cache with a lot of trackables actions

### ISSUE THREAD

**Issue by kumy** [cgeo/cgeo#5133]
When logging a cache, I set most trackables in my inventory as visited. As I have a huge amount of GK trackables, process is quite long and the only feedback returned to user is text "Sending log...".

I would be great if we can display current progress for each step of posting log, with some counter.

Looking at the code, steps could be:
1. Posting cache log
2. Posting Tweet
3. Posting GCVote
4. (loop) Posting Trackables `<Connector name>` X/Y (with X increasing)
5. Posting Image

Any objection?

**Comment by Lineflyer** [comment]
Nice addon, but:
- When looking at GC-trackable we do not have any progress as they are implicitly logged while sending the log
- Aren't we doing some of the listed things in parallel already?
- Given a fast network connection most items of the list will hardly be visible as they are done too fast
- Wouldn't it be more useful to tell the user which part has failed (if any) at the end of the network operation instead of showing the progress 

I don't know the effort is to implement this, but IMHO the effort should instead be used for other things (like enhancements to the functionality itself).

**Comment by kumy** [comment]
@Lineflyer thanks for you're feedback.

Currently, actions are sequential. (for reference: https://github.com/cgeo/cgeo/blob/master/main/src/cgeo/geocaching/LogCacheActivity.java#L461-L525)
GC-trackables are logged at "stage 1".
I log ~200 GK and it take about 4 minutes to complete (an eternity!).

I have a working code, so the effort is almost done, and mainly, I'm interested in such a feature ;)

However, you touched a good point `tell the user which part has failed (if any)` (shouldn't we track this as another issue?)

**Comment by Lineflyer** [comment]
> However, you touched a good point tell the user which part has failed (if any) (shouldn't we track this as another issue?)

Sure.
